README.md

```diff
@@ -193,6 +202,28 @@ The autograder failed to grade your assignment.
 Message from the autograder: Request could not be authenticated. Ensure that your username, password, and course are properly set.
 ```
 
+##### Submitting an Assignment Late
+
+If you are submitting an assignment late, the autograder requires confirmation in order to grade your submission.
+This helps users avoid situations where they accidentally submit an assignment late or submit to the wrong assignment.
+Users must add the `--allow-late` flag to the normal submission command when they want to submit an assignment past the due date.
+
+Here is a sample output when trying to submit a late assignment without the flag:
+```
+--- Message from Autograder ---
+Attempting to submit assignment (<ASSIGNMENT NAME>) late without 'allow late'. It was due on <DUE DATE> (which was <AMOUNT OF TIME LATE> ago). Include the 'allow late' flag to submit an assignment late. See your interface's help section for more information.
```

Comment:
See the other examples, which use actual (fake) values.
Note that this will change because of changes in the other PR.